python/ray/tune/tune.py

```diff
@@ -295,17 +295,25 @@ def run(
         restore: Path to checkpoint. Only makes sense to set if
             running 1 trial. Defaults to None.
         server_port: Port number for launching TuneServer.
-        resume: One of "LOCAL", "REMOTE", "PROMPT", "ERRORED_ONLY", "AUTO",
-            or bool. "LOCAL"/True restores the checkpoint from the
+        resume: One of [True, False, "LOCAL", "REMOTE", "PROMPT", "AUTO"]. Can
+            be suffixed with one or more of ["+ERRORED", "+ERRORED_ONLY",
+            "+RESTART_ERRORED", "+RESTART_ERRORED_ONLY"] (e.g. ``AUTO+ERRORED``).
+            "LOCAL"/True restores the checkpoint from the
```

Comment:
`tune.run` is still external facing at this point right?
My understanding is user should not bother with using tune.run with "xxx+yyy" sort of configurations?
so question is should we still keep the docstring unchanged?

Reply:
or should we consider just having an internal only argument for resume config for `tune.run` - to spare the trouble of "xxx+yyy" and then parse it later?

Reply:
I actually think we should just support the `+suffix` strings - users have been asking for e.g. REMOTE+ERRORED in the past, and this way they can also do this with the tune.run API.
In this case I also think it's fine to keep it as there is a clear conversion between the strings and the internal resume config - thus if we decide to later change the tune.run() API to use a config, we can easily deprecate strings (which we would have to do anyways).